runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt

```diff
@@ -76,10 +76,10 @@ abstract class AbstractDokkaTask : DefaultTask() {
     }
 
     @Classpath
-    val plugins: Configuration = project.maybeCreateDokkaPluginConfiguration(name)
+    val plugins: ConfigurableFileCollection = project.objects.fileCollection()
```

Comment:
You can drop explicit initialization, just make this
`abstract val plugins: ConfigurableFileCollection` and Gradle will initialize this for you.

Reply:
Thanks, I'm aware, I just left it as it currently is to align with the rest of the parameters which still use manual initialisation.